example/spec/parallel/testAPI/bulkDeleteSuccessSpec.js

```diff
@@ -176,21 +180,31 @@ describe('POST /granules/bulkDelete', () => {
 
     afterAll(async () => {
       // Must delete rules and executions before deleting associated collection and provider
-      await deleteRule({ prefix, ruleName: get(ingestGranuleRule, 'name') });
-      await deleteExecution({ prefix: config.stackName, executionArn: ingestGranuleExecution1Arn });
+      await deleteRule(
+        { prefix, ruleName: get(ingestGranuleRule, 'name') }
+      );
+      await deleteExecution(
+        { prefix: config.stackName, executionArn: ingestGranuleExecution1Arn }
+      );
 
       if (postBulkDeleteBody.id) {
-        await deleteAsyncOperation({ prefix: config.stackName, asyncOperationId: postBulkDeleteBody.id });
+        await deleteAsyncOperation(
+          { prefix: config.stackName, asyncOperationId: postBulkDeleteBody.id }
+        );
       }
 
       await pAll(
         [
-          () => deleteProvider({ prefix, providerId: get(provider, 'id') }),
-          () => deleteCollection({
-            prefix,
-            collectionName: get(collection, 'name'),
-            collectionVersion: get(collection, 'version'),
-          }),
+          () => deleteProvider(
+            { prefix, providerId: get(provider, 'id') }
+          ),
+          () => deleteCollection(
+            {
+              prefix,
+              collectionName: get(collection, 'name'),
+              collectionVersion: get(collection, 'version'),
+            }
+          ),
```

Comment:
It looks like these changes are all spacing/line-break-related. I think it does look cleaner here. Should we make a couple lint rules so you don't have to do this again in the next PR? 😄

Reply:
or maybe there's not a way to not make it a rule without breaking all the things but we can set our editors the same way at least 🤷

Reply:
Just noticed this comment - I think that using an opinionated linter (e.g. prettier) is the way to go here.